docs/java-helloworld.md

```diff
@@ -47,27 +47,27 @@ Once Gradle has finished scaffolding you will need to customize the project depe
 - `io.temporal:temporal-sdk` enables communication with the Temporal server.
 - `ch.qos.logback:logback-classic` will ensure that there is a logger to bind to within the SDK and prevent a default logger warning message.
 
-A "refresh" icon will appear on the screen, click it to load the changes. Gradle will rebuild with the dependencies.
-
 To limit the logging output from the SDK, within src/main/resources/ create a logback.xml file and paste in the following XML:
 
 <!--SNIPSTART hello-world-project-template-java-logback-dependency-configuration-->
 <!--SNIPEND-->
 
-## ![](/img/docs/apps.png) &nbsp;&nbsp; "Hello World!" app
+If you are editing the files in IntelliJ, a "refresh" icon will appear on the screen, click it to load the changes. Gradle will rebuild with the dependencies. Otherwise you can run `./gradlew build` from the root of the project again.
```

Comment:
```suggestion
If you are editing the files in IntelliJ, a "refresh" icon will appear on the screen. Click it to load the changes. Gradle will rebuild with the dependencies. Otherwise you can run `./gradlew build` from the root of the project again.
```